Dockerfile

```diff
@@ -1,7 +1,14 @@
 FROM node:14-alpine as build-base
 
-RUN apk add --update-cache git python make g++ && rm -rf /var/cache/apk/*
-RUN git config --global url."https://github.com/".insteadOf 'ssh://git@github.com/'
+RUN apk add --update-cache git python make g++ openssh-client && rm -rf /var/cache/apk/*
```

Comment:
The ch-sdk-node library is private, so developer GitHub creds need to be copied into the Docker container for now. Ideally, a generic Docker GitHub user would be available, but this will unblock us for now. I'm using SSH to fetch the git repos now.

Steps on how to set this up have been added to the README - https://github.com/companieshouse/dissolution-web/pull/18/files#diff-04c6e90faac2675aa89e2176d2eec7d8R38